porcupine/plugins/markdown.py

```diff
@@ -0,0 +1,56 @@
+"""If configuration says so, insert spaces when the tab key is pressed."""
```

Comment:
Copy/pasta from another plugin. This must be correct/reasonable, because it shows up in the plugin manager.

Reply:
Makes sense, oversight on my part.

Reply:
Fixed with 8b3e3df0586385815e483df3cd05a04f76809e65